<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictionaryRow</class>
 <widget class="QWidget" name="DictionaryRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QLabel" name="question">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="answer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>266</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="trueAnswer">
       <property name="text">
        <string>Полное совпадение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="semiTrueAnswer">
       <property name="text">
        <string>Неполное совпадение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="falseAnswer">
       <property name="text">
        <string>Ответ неправильный</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
